test/games_test.rb

```diff
@@ -19,50 +20,51 @@ def test_events_starts_empty
     assert_equal [], games.events
   end
 
-  def test_it_can_add_events
+   def test_it_can_add_events
     curling = Event.new("Curling", [24, 30, 18, 20, 41])
     ring_toss = Event.new("Ring Toss", [23, 22, 29, 18, 30])
     games = Games.new(2017)
     games.add_event(curling)
     games.add_event(ring_toss)
     assert_equal [curling, ring_toss], games.events
-  end
+   end
 
   def test_it_can_create_headers
     games = Games.new(2017)
-    expected = "Event          Max Age             Min Age             Average Age         StdDev Age"
+     expected = "Event          Max Age             Min Age             Average Age         StdDev Age"
     assert_equal expected, games.headers
   end
 
-  def test_it_can_create_a_single_event_summary
+   def test_it_can_create_a_single_event_summary
     curling = Event.new("Curling", [24, 30, 18, 20, 41])
-    games = Games.new(2017)
-    expected = "Curling        41                  18                  26.6                8.28"
-    assert_equal expected, games.event_summary(curling)
+     games = Games.new(2017)
+     expected = "Curling        41                  18                  26.6                8.28"
+     assert_equal expected, games.event_summary(curling)
```

Comment:
Minor thing, but betting that you didn't mean to commit these changes. Careful when you're adding files to make sure that you're only adding things that you expected to change.